### PR TITLE
Update README.md 1st draft

### PR DESCRIPTION
**Pull request by clc80**

### CODE REVIEW

**Comment by vuinguyen** [comment]
To implement issue #53

**Review by devanshimodha — APPROVED** [review]

**Review by vuinguyen — CHANGES_REQUESTED** [review]
This looks good! 

One suggestion:
There is a readme.template file here:
coding-projects/resources/project-readme-template.md

If you're going to create the template in this README.md file instead, I would recommend removing the project-readme-template.mid file then to avoid confusion.

Thanks for working on this issue @clc80 !

File: coding-projects/README.md [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+
+3. **Usage**:
+   - [Provide detailed instructions on how to use the project, including commands, functionalities, or any necessary guides for contributors.]
+
+4. **Contribution Guidelines**:
+   - Fork this repository.
+   - Create a new branch: `git checkout -b feature/your-feature`.
+   - Commit your changes: `git commit -am 'Add new feature'`.
+   - Push to the branch: `git push origin feature/your-feature`.
+   - Submit a pull request.
```
Comment (vuinguyen) [inline coding-projects/README.md:20]: I would add to this:
Short summary of contributions here:
.....

More detailed instructions for contributing and the Git workflow can be found in the Resources folder.

**Review by vuinguyen — APPROVED** [review]
Since Claudia is taking a break, I think the easier thing to do would be to approve and merge this PR. Then, I can submit additional changes through a separate PR.

**Pull request merged**